glotaran/parameter/parameter_group.py

```diff
@@ -202,10 +164,10 @@ def from_csv(cls, filepath: str, delimiter: str = "\t"):
                 p = Parameter(label=label.pop())
                 p.value = df["value"][i]
                 p.stderr = df["stderr"][i]
-                p.min = df["min"][i]
-                p.max = df["max"][i]
+                p.minimum = df["min"][i]
```

Comment:
Note: we're not changing the key for csv with this PR, hence renaming the key for internal use only.